### PR TITLE
Update issue templates

### LINKED ISSUE #22: Introduce "plain" issue template for people who know what they're doing
The current templates require people to fill out the form, but for some issues (like this) none of the templates apply.

A plain template should cover these.

Templates are located under

https://github.com/pymc-devs/pytensor/tree/main/.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by ricardoV94**
Mirrors https://github.com/pymc-devs/pymc/pull/6345 and closes #22

### CODE REVIEW

**Comment by michaelosthege** [comment]
@ricardoV94 there is already a `plain.yml` since #42

taking a look at your changes now

File: .github/ISSUE_TEMPLATE/developer.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Developer issue
+description: This template is for developers only!
+
+body:
+- type: textarea
```
Comment (michaelosthege) [inline .github/ISSUE_TEMPLATE/developer.yml:5]: Merge this with the `plain.yml` issue?

Comment (ricardoV94) [reply .github/ISSUE_TEMPLATE/developer.yml:5]: Updated, is this what you meant?

**Comment by ricardoV94** [comment]
Combined the plain and dev issues.

**Review by michaelosthege — APPROVED** [review]

**Pull request merged**